river/command/map.zig

```diff
@@ -46,9 +46,13 @@ pub fn map(
 
     if (optionals.release and optionals.repeat) return Error.ConflictingOptions;
 
-    const mode_id = try modeNameToId(allocator, args[1 + offset], out);
-    const modifiers = try parseModifiers(allocator, args[2 + offset], out);
-    const keysym = try parseKeysym(allocator, args[3 + offset], out);
+    const mode_name = args[1 + offset];
+    const modifiers_name = args[2 + offset];
+    const keysym_name = args[3 + offset];
```

Comment:
We use `raw` elsewhere in the codebase to describe unprocessed user input like this, I think it would be nice to do the same here.

Reply:
It's not multiple keys though, it's a single keysym.